daggerutil/fileexists.go

```diff
@@ -0,0 +1,16 @@
+package daggerutil
+
+import (
+	"context"
+
+	"dagger.io/dagger"
+)
+
+func FileExists(ctx context.Context, dir *dagger.Directory, path string) bool {
+	_, err := dir.File(path).Contents(ctx)
+	if err == nil {
+		return true
```

Comment:
I'd suggest inverting this condition and logging the error (just in case it's something unrelated to the file not actually existing).

Reply:
Thanks for this suggestion @aangelisc implemented in https://github.com/grafana/grafana-build/pull/294/commits/b1e6cc3bf1f883a2495330b6670d5fbfb843d392